Add `Proxy` docs (#26)
Bisonai/orakl-docs: en/node-operators-guide/cli/proxy.md

---
description: List, Insert And Remove Orakl Network Proxies
---

# Proxy

> The **Orakl Network ** state can store information about the multiple proxies. Proxies are can be utilized to fetch data by making requests through defined proxy configurations

Reviewer: ```suggestion
The **Orakl Network ** state can store information about the multiple proxies. Proxies are can be utilized to fetch data by making requests through defined proxy configurations.
```

The **Orakl Network CLI** provides commands to

* [List Proxies](proxy.md#list-proxies)
* [Insert New Proxy](proxy.md#insert-new-proxy)
* [Remove Proxy Specified By `id`](proxy.md#remove-proxy-specified-by-id)

### List Proxies

```sh
orakl-cli proxy list
```

```json
[
  { id: '1', protocol: 'http', host: '127.0.0.1', port: 80},
  { id: '1', protocol: 'http', host: '127.0.0.1', port: 88'},
]
```

### Insert New Proxy

> If there are not proxies set, we can add a new proxy using `proxy insert` command of the **Orakl Network CLI**.

Reviewer: ```suggestion
Proxies can be added with a `proxy insert` command. A unique proxy can be added only once.
```
